src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -215,6 +227,23 @@ private static void PromptRequiredProperties<T>(T manifest, VersionManifest vers
             }
         }
 
+        private static void PromptOptionalProperties<T>(T manifest)
+        {
+            var properties = manifest.GetType().GetProperties().ToList();
+            var optionalProperties = properties.Where(p => p.GetCustomAttribute<RequiredAttribute>() == null).ToList();
+
+            foreach (var property in optionalProperties)
+            {
+                if (property.PropertyType == typeof(string) || property.Name == nameof(DefaultLocaleManifest.Tags))

```

Comment:
Do you still need this filter at all? If this is to avoid prompting for additionalProperties, you could change 233 to only include properties with the JsonProperty attribute too

Reply:
Added additional filter clause to filter out non-jsonproperty attributes